Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs

```diff
@@ -150,6 +152,15 @@ public override string ToString()
             return $"{nameof(Salt)}: {Salt}, {nameof(EnablePost)}: {EnablePost}, {nameof(EnableConsoleLogging)}: {EnableConsoleLogging}, {nameof(IgnoreSessionCooldown)}: {IgnoreSessionCooldown}, {nameof(NotificationMode)}: {NotificationMode}, {nameof(EnableManualSessionHandling)}: {EnableManualSessionHandling}, {nameof(SessionDuration)}: {SessionDuration}, {nameof(EventQueueThreshold)}: {EventQueueThreshold}, {nameof(StoredRequestLimit)}: {StoredRequestLimit}, {nameof(TotalBreadcrumbsAllowed)}: {TotalBreadcrumbsAllowed}, {nameof(EnableAutomaticCrashReporting)}: {EnableAutomaticCrashReporting}";
         }
 
+        /// <summary>
+        ///Disabled the automatic session tracking.
+        /// </summary>
+        public void DisableAutomaticSessionTracking()
+        {
+            IsAutomaticSessionTrackingDisabled = true;
+            Debug.Log("[Countly][CountlyConfiguration] Automatic session tracking disabled!");
```

Comment:
where should it be?
We can print this where the first time 'session begins' calls automaically.

Reply:
That information was already in the ticket:
"In the module an appropriate log should be printed when this is done." 
So that means in the session module when it is being initialised.